make down command not working

**Describe the bug**
'make down' command not working

**To Reproduce**
```
elastdocker$ make down
ERROR: The Compose file './docker-compose.tools.yml' is invalid because:
services.rubban.environment.RUBBAN_AUTOINDEXPATTERN_ENABLED contains true, which is an invalid type, it should be a string, number, or a null
Makefile:61: recipe for target 'down' failed
make: *** [down] Error 1
```

**Solution:**
edit `./docker-compose.tools.yml`:
currently - `RUBBAN_AUTOINDEXPATTERN_ENABLED: true`
should be - `RUBBAN_AUTOINDEXPATTERN_ENABLED: 'true'`


BR,
Vasily.

--- a/docker-compose.tools.yml
+++ b/docker-compose.tools.yml
@@ -11,6 +11,6 @@ services:
       RUBBAN_REFRESHINDEXPATTERN_ENABLED: 'true'
       RUBBAN_REFRESHINDEXPATTERN_SCHEDULE: '*/5 * * * *'
       RUBBAN_REFRESHINDEXPATTERN_PATTERNS: '*'
-      RUBBAN_AUTOINDEXPATTERN_ENABLED: true
+      RUBBAN_AUTOINDEXPATTERN_ENABLED: 'true'
       RUBBAN_AUTOINDEXPATTERN_SCHEDULE: '*/5 * * * *'
       RUBBAN_AUTOINDEXPATTERN_GENERALPATTERNS: '[{"pattern":"filebeat?","timeFieldName":"@timestamp"},{"pattern":"logstash?","timeFieldName":"@timestamp"}]'

